alfalfa_worker/jobs/openstudio/full_run.py

```diff
@@ -0,0 +1,24 @@
+
+import subprocess
+from datetime import datetime
+
+import pytz
+
+from alfalfa_worker.lib.alfalfa_connections_base import AlfalfaConnectionsBase
+from alfalfa_worker.lib.job import Job
+
+
+class FullRun(AlfalfaConnectionsBase, Job):
+
+    def __init__(self, run_id) -> None:
+        self.run = self.checkout_run(run_id)
+
+    def exec(self) -> None:
+        osws = self.run.glob("**/*.osw")
+        for oswpath in osws:
+            subprocess.call(['openstudio', 'run', "-w", oswpath])
+
+        time = str(datetime.now(tz=pytz.UTC))
```

Comment:
eek, timezones

Reply:
yeah... I just copied the pattern we were using elsewhere. If we want to address that it is definitely a possibility